### ISSUE TITLE
Disable column attribute does not update table

### ISSUE THREAD

**Issue by dotpointer** [Neovici/cosmoz-omnitable#204]
Changing the `disable` attribute on a cosmoz-omnitable-column does not show nor hide the column in the table when the table has already been rendered. The attribute only works when reloading the page. Related issue to this is #19628.

**Comment by dotpointer** [comment]
Or maybe the fault is somewhere else. I get this error in the console when the columns should have been updated:

```
templatize.html:562 Uncaught Error: A <template> can only be templatized once
    at Object.templatize (templatize.html:562)
    at HTMLElement.templatize (templatizer-behavior.html:106)
    at HTMLElement._ensureTemplatized (cosmoz-omnitable-templatizer.html:30)
    at HTMLElement.getInstance (cosmoz-omnitable-templatizer.html:66)
    at HTMLElement._getTemplateInstance (cosmoz-omnitable-item-row.html:54)
    at HTMLElement._addElements (cosmoz-omnitable-repeater-behavior.html:171)
    at HTMLElement._columnsChanged (cosmoz-omnitable-repeater-behavior.html:143)
    at Object.runMethodEffect [as fn] (property-effects.html:904)
    at runEffectsForProperty (property-effects.html:168)
    at runEffects (property-effects.html:130)
```

**Comment by megheaiulian** [comment]
@dotpointer Please have a look at the related PR.

**Comment by dotpointer** [comment]
I have not found any reason why this was closed _without confirming the issue as solved_.

The issue is still present, tested both with local frontend and beta backend, also tested directly against beta.

Error is the same when switching branches and having the table displayed. Instead of showing/hiding columns by `disabled` I get the following error in the console:

```
templatize.html:562 Uncaught Error: A <template> can only be templatized once
    at Object.templatize (templatize.html:562)
    at HTMLElement.templatize (templatizer-behavior.html:106)
    at HTMLElement._ensureTemplatized (cosmoz-omnitable-templatizer.html:30)
    at HTMLElement.getInstance (cosmoz-omnitable-templatizer.html:66)
    at HTMLElement._getTemplateInstance (cosmoz-omnitable-item-row.html:54)
    at HTMLElement._addElements (cosmoz-omnitable-repeater-behavior.html:171)
    at HTMLElement._columnsChanged (cosmoz-omnitable-repeater-behavior.html:134)
    at Object.runMethodEffect [as fn] (property-effects.html:904)
    at runEffectsForProperty (property-effects.html:168)
    at runEffects (property-effects.html:130)
```

**Comment by nomego** [comment]
Issue was closed automatically because the PR said "fixes" and was merged.
Have you wiped bower_components and run yarn?
@megheaiulian more issues to fix?

**Comment by dotpointer** [comment]
Yes I removed the whole bower_components directory and reinstalled, also tested on alpha.

**Comment by plequang** [comment]
Issue was closed by PR #208, but reopening because not completely solved.
As explained in the PR:

> If  the table is not visible when changing the `disabled` property of a column, then `_updateColumns` does nothing, causing the table to potentially display columns it shouldn't.
> Current case is when we display a "loading"dialog over the table while fetching new data, and setting the new data before the dialog is removed.

Not sure this can be solved in omnitable, but maybe it might be investigated

**Comment by nomego** [comment]
Is this solved by #241 ?

**Comment by dotpointer** [comment]
> Is this solved by #241 ?

Not sure if that made it, but I tested it in the accounting dimension values tab as mentioned in #19628 by attaching a value forwarding function on the disabled attribute and then I switched node. It disabled and enabled the column as intended. Closing this.
